images/pod/Dockerfile.centos7

```diff
@@ -22,6 +22,6 @@ RUN INSTALL_PKGS="origin-pod" && \
     ln -s /usr/bin/pod /pod
```

Comment:
In what cases do we want to carry diffs between `Dockerfile` and `Dockerfile.centos7` ?